From 4d63c2c11039794ed61412021901078b15b48116 Mon Sep 17 00:00:00 2001
From: Chris Nauroth <cnauroth@apache.org>
Date: Fri, 7 Oct 2016 11:41:19 -0700
Subject: [PATCH 2145/2748] HADOOP-13692. hadoop-aws should declare explicit
 dependency on Jackson 2 jars to prevent classpath
 conflicts. Contributed by Chris Nauroth.

(cherry picked from commit 69620f955997250d1b543d86d4907ee50218152a)

Change-Id: I9c88ae97a4ef1bb031f365279d84d08daa06b833
---
 hadoop-tools/hadoop-aws/pom.xml |   12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/hadoop-tools/hadoop-aws/pom.xml b/hadoop-tools/hadoop-aws/pom.xml
index 7340008..c2ce16f 100644
--- a/hadoop-tools/hadoop-aws/pom.xml
+++ b/hadoop-tools/hadoop-aws/pom.xml
@@ -263,6 +263,18 @@
       -->
     </dependency>
     <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
+    </dependency>
+    <dependency>
       <groupId>joda-time</groupId>
       <artifactId>joda-time</artifactId>
     </dependency>
-- 
1.7.9.5

